# a vpPoint

int32 i
int32 j
float64 X
float64 Y
float64 Z

